src/test/Fake.ExpectoSupport/ExpectoHelpers.fs

```diff
@@ -5,86 +5,6 @@ open System.Threading
 open System.Threading.Tasks
 
 module ExpectoHelpers =
-
-    let inline internal commaString (i:int) = i.ToString("#,##0")
-    // because of https://github.com/haf/expecto/issues/367
```

Comment:
Just FYI, if everything works. OK, but I doubt that this issue has been solved. So if you ever encounter a build 'hanging', this might be the root cause. (I'd leave it as is and see if bad things happen)

Reply:
For Expecto, I read the [issue](https://github.com/haf/expecto/issues/367) that has caused the inlining of the Expecto printer in FAKE. But as I understand this was caused in Travis-CI builds, and now we are using GitHub for everything and didn't encounter it till now 😅. So, we can try and see. If anything hangs I'll inline the printer as before.

Reply:
@matthid It seems that Expecto tests are failing on the `release/next` branch after the upgrade to .Net 6.0.101 from the preview version. In the PR, the build succeeded but now Expecto gives [timeout on unit tests](https://github.com/fsprojects/FAKE/runs/4880781893?check_suite_focus=true#step:10:3814). We are still using the custom printer on the `release/next` branch. I tried to increase the timeout to 30 minutes and re-run the build multiple times but no luck. I'll investigate more on this, not sure if the update to Expecto 9 and the removal of the custom printer would solve it which are done in this PR. But we will not know that until this PR got fixed. 

I just wanted to give an update on this. Will try to look more and see what I got.

Thanks